### PR TITLE
Add postbuild dependencies check

### PR DESCRIPTION
**Pull request by gquadrati**
In order to avoid missing dependencies after _yarn install --prod_ command, a postbuild check has been added to prompt all dependencies not listed in package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "license": "MIT",
   "scripts": {
     "prebuild": "shx rm -rf dist && yarn generate",
+    "postbuild": "npx dependency-check package.json --missing ./dist/**/*.js",
```
Comment (balanza) [inline package.json:7]: Avoid the use of dev dependencies

```suggestion
    "postbuild": "npx dependency-check package.json --no-dev --missing ./dist/**/*.js",
```

**Review by balanza — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gquadrati) [reply package.json]: using _--no-dev_ we see another dependency: _danger-plugin-digitalcitizenship_ 
This is used within Dangerfile.js
Is it correct to have this file in _dist/_ fodler?

File: tsconfig.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "__mocks__",
     "dist",
     "node_modules",
-    "**/__tests__/*"
+    "**/__tests__/*",
+    "Dangerfile.ts"
```
Comment (gquadrati) [inline tsconfig.json:16]: @balanza  do you think is correct to exclude this file from tsconfig.json?

**Comment by balanza** [comment]
Ok, let's say we are fine with this check. We now have a failing check, and we must fix the build ;)

**Comment by gquadrati** [comment]
> Ok, let's say we are fine with this check. We now have a failing check, and we must fix the build ;)

I wanted to try the Dangerfile before adding dependencies :) I'm doing it

**Review by balanza — APPROVED** [review]

**Pull request merged**